pkg/controller/manager/controller.go

```diff
@@ -34,7 +34,11 @@ const (
 	ControllerName = "ScyllaManagerController"
 	// maxSyncDuration enforces preemption. Do not raise the value! Controllers shouldn't actively wait,
 	// but rather use the queue.
-	maxSyncDuration = 30 * time.Second
+	// Unfortunately, Scylla Manager calls are synchronous, internally retried and can take ages.
+	// Contrary to what it should be, this needs to be quite high.
+	// Given we reconcile tasks individually with an API call,
+	// this may not be enough for N task but should eventually make it.
+	maxSyncDuration = 2 * time.Minute
```

Comment:
does that imply that individual tasks can take longer than 30 seconds? Otherwise requeuing on our side should reach the same result?

Reply:
we do more then 1 manager API call per sync loop

Reply:
it has been failing permanently on my local setup so this definitely helps. We can file a new one if there is still something left and it flakes again.